src/remodal.js

```diff
@@ -431,6 +422,22 @@
   }
 
   /**
+   * Generates a string separated by dashes and prefixed with NAMESPACE
+   * @private
+   * @param str, str, ...
+   * @returns str
+   */
+  function namespaced() {
+    var result = NAMESPACE;
+
+    for (var i = 0; i < arguments.length; ++i) {
```

Comment:
Well, `length` is an argument on `arguments` object, so calling it has **zero** extra cost and there's no need for caching it locally.